<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="FBTestSuite" duration-ms="5693" started-at="2018-12-04T10:22:03Z" finished-at="2018-12-04T10:22:08Z">
    <groups>
    </groups>
    <test name="FunctionalTestSuite" duration-ms="5693" started-at="2018-12-04T10:22:03Z" finished-at="2018-12-04T10:22:08Z">
      <class name="test.FbTest">
        <test-method status="PASS" signature="PreSetup()[pri:0, instance:test.FbTest@3a82f6ef]" name="PreSetup" is-config="true" duration-ms="12670" started-at="2018-12-04T15:51:50Z" finished-at="2018-12-04T15:52:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreSetup -->
        <test-method status="FAIL" signature="testing()[pri:0, instance:test.FbTest@3a82f6ef]" name="testing" duration-ms="5389" started-at="2018-12-04T15:52:03Z" finished-at="2018-12-04T15:52:08Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Pages.FbHomePage.<init>(FbHomePage.java:15)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.lang.Class.newInstance(Unknown Source)
	at org.openqa.selenium.support.PageFactory.instantiatePage(PageFactory.java:131)
	at org.openqa.selenium.support.PageFactory.initElements(PageFactory.java:64)
	at test.FbTest.testing(FbTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testing -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:test.FbTest@3a82f6ef]" name="TearDown" is-config="true" duration-ms="1939" started-at="2018-12-04T15:52:09Z" finished-at="2018-12-04T15:52:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- test.FbTest -->
    </test> <!-- FunctionalTestSuite -->
  </suite> <!-- FBTestSuite -->
</testng-results>
